storage/aws/antispam/aws_test.go

```diff
@@ -108,6 +107,61 @@ func TestAntispam(t *testing.T) {
 	}
 }
 
+func TestAntispamPushback(t *testing.T) {
```

Comment:
You might want to describe what behavior you're implementing here, such that if we ever change some functions and have to change this test, we know what we want to reproduce exactly.

Reply:
I've renamed the tests to be clearer about what they're testing, and expanded the comment at the bottom where they check `pushBack`.